homeassistant/components/minecraft_server/entity.py

```diff
@@ -17,14 +21,15 @@ class MinecraftServerEntity(Entity):
     def __init__(
         self,
         server: MinecraftServer,
-        type_name: str,
+        entity_type: str,
         icon: str,
         device_class: str | None,
     ) -> None:
         """Initialize base entity."""
         self._server = server
         self._attr_icon = icon
-        self._attr_unique_id = f"{self._server.unique_id}-{type_name}"
+        self._attr_unique_id = f"{self._server.unique_id}-{entity_type}"
+        _LOGGER.debug("Unique ID of entity type %s: %s", entity_type, self.unique_id)
```

Comment:
Do we sill need this?

Reply:
I'm not sure. The new unique IDs are printed as debug logs during migration. But migration only happens once and then we don't have access to the new unique IDs. That's why I added them also here. 
I could remove them, if you think they don't make sense here?